<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sdashb</class>
 <widget class="QMainWindow" name="Sdashb">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>800</height>
   </rect>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="StudentName">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>231</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Product Sans</family>
      <pointsize>15</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="Welcome">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>111</width>
      <height>33</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>6</width>
      <height>5</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>9</width>
      <height>10</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Product Sans</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Welcome,</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>511</width>
      <height>991</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(242, 242, 242);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>470</y>
       <width>381</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);
border-radius: 10px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>344</width>
        <height>195</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>391</width>
       <height>121</height>
      </rect>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(218, 218, 218);
border-radius: 10px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe MDL2 Assets</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Images.qrc">:/new/prefix1/ProfileIcon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="name">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Louis George Cafe</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Name: </string>
      </property>
     </widget>
     <widget class="QLabel" name="studentNum">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>321</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Louis George Cafe</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Student #:</string>
      </property>
     </widget>
     <widget class="QLabel" name="email">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>321</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Louis George Cafe</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Coolvetica</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar {
    border-radius: 3px;               /* Border radius to match the other elements */
    text-align: center;               /* Center align the text */
    color: #ffffff;                   /* Text color (white) */
    background-color: #d4d4d4;        /* Background color of the QProgressBar (neutral gray) */
}

QProgressBar::chunk {
    background-color: #337ab7;        /* Progress bar color (blue) matching QPushButton */
    border-radius: 3px;               /* Rounded corners for the progress bar */
}


QProgressBar::chunk:hover {
    background-color: #285e8e;        /* Progress bar color on hover */
}
</string>
      </property>
      <property name="value">
       <number>15</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Coolvetica</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Rating Progress:</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>391</width>
       <height>191</height>
      </rect>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(218, 218, 218);
border-radius: 10px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="EvaluateButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>140</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Product Sans Light</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #337ab7;  /* Button background color (blue) */
    color: #ffffff;             /* Text color (white) */
	border-radius: 3px;
}
QPushButton:hover{
	background-color: #285e8e;
}</string>
      </property>
      <property name="text">
       <string>Evaluate</string>
      </property>
     </widget>
     <widget class="QLabel" name="AcadYear">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>361</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Product Sans Light</family>
        <pointsize>18</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Academic Year: 2024-2025&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="StatusBar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>221</width>
        <height>33</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Product Sans</family>
        <pointsize>13</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Evaluation Status:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>90</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Product Sans</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: green;</string>
      </property>
      <property name="text">
       <string>ONGOING</string>
      </property>
     </widget>
     <widget class="QPushButton" name="EditButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Product Sans Light</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #337ab7;  /* Button background color (blue) */
    color: #ffffff;             /* Text color (white) */
	border-radius: 3px;
}
QPushButton:hover{
	background-color: #285e8e;
}</string>
      </property>
      <property name="text">
       <string>Edit Profile</string>
      </property>
     </widget>
     <widget class="QLabel" name="AcadYear_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Product Sans Light</family>
        <pointsize>18</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Second Semester&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>90</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Product Sans</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: red;</string>
      </property>
      <property name="text">
       <string>ENDED</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>90</y>
        <width>16</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Product Sans</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: BLACK;</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>391</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="logoutButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>80</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #337ab7;  /* Button background color (blue) */
    color: #ffffff;             /* Text color (white) */
	border-radius: 3px;
}
QPushButton:hover{
	background-color: #285e8e;
}
</string>
     </property>
     <property name="text">
      <string>Log Out</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>-30</x>
       <y>0</y>
       <width>461</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(55, 122, 183);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Product Sans Black</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>DASHBOARD</string>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>frame</zorder>
   <zorder>StudentName</zorder>
   <zorder>Welcome</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
